_sources/Dictionaries/Dictionarymethods.rst

```diff
@@ -192,44 +236,34 @@ a key, return 0 (instead of None).
             total = total + mydict[akey]
       print(total)
 
-.. activecode:: ac10_3_6
-   :language: python
-   :autograde: unittest
-   :practice: T
-
-   **5.** Every four years, the summer Olympics are held in a different country. Add a key-value pair to the dictionary ``places`` that reflects that the 2016 Olympics were held in Brazil. Do not rewrite the entire dictionary to do this!
-   ~~~~
+.. tabbed:: tabbed_ac10_3_7
 
-   places = {"Australia":2000, "Greece":2004, "China":2008, "England":2012}
+    .. tab:: Question
 
-   =====
+        **5.** We have a dictionary of the specific events that Italy has won medals in and the number of medals they have won for each event. Assign to the variable ``events`` a list of the keys from the dictionary ``medal_events``. Do not hard code this.
```

Comment:
We need to keep the instructions inside the directive so they will stay together in the case where an instructor assigns this question.

Reply:
Fixed.